src/languageservice/services/yamlCompletion.ts

```diff
@@ -260,7 +260,7 @@ export class YAMLCompletion extends JSONCompletion {
     addValue: boolean,
     separatorAfter: string,
     collector: CompletionsCollector,
-    document
+    document: TextDocument
```

Comment:
Did you know that `TextDocument` is deprecated?

Reply:
Yes, but we use it across all project.